scene/gui/control.cpp

```diff
@@ -2419,7 +2419,7 @@ void Control::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_font_changed"), &Control::_font_changed);
 
 	BIND_VMETHOD(MethodInfo("_gui_input", PropertyInfo(Variant::INPUT_EVENT, "event")));
-	BIND_VMETHOD(MethodInfo(Variant::VECTOR2, "get_minimum_size"));
+	BIND_VMETHOD(MethodInfo(Variant::VECTOR2, "_get_minimum_size"));
```

Comment:
Call the new method, what do you mean? And why? All other virtual methods don't use SceneStringName as well